web/src/main/java/org/fao/geonet/monitor/link/LinkMonitorInterface.java

```diff
@@ -0,0 +1,44 @@
+package org.fao.geonet.monitor.link;
+
+import jeeves.interfaces.Logger;
+import jeeves.server.resources.ResourceManager;
+import org.fao.geonet.GeonetContext;
+
+public class LinkMonitorInterface {
```

Comment:
Umn, OK, fair enough. Would you like it as `LinkMonitorAbstractClass`? `LinkMonitorBase`? I'm not so sure.